package io.exonym.lib.exceptions;

public class AlreadyAuthException extends Exception {
}
